Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
@robisim74 @FabianGosebrink By separating the polyfills, the app bundle is reduced in size by a ~100KB 
polyfills is only 2KB

Overall size is reduced, but the application has 3 files to download instead of 1
Now:
vendor : 311KB
app: 589KB

Separated:
vendor : 311KB
app: 493KB
polyfills: 2KB

What do you think?